requirements.txt

```diff
@@ -19,12 +19,11 @@ plotnine==0.7.0
 protobuf==3.7.0
 python-dateutil==2.8.0
 pytz==2018.9
-scikit-learn==0.23.2
```

Comment:
Pretty sure we need this in evaluation? https://github.com/google-research/proteinfer/blob/master/evaluation.py#L30

Reply:
thanks, fixed